### ISSUE TITLE
Clarify ordering of k8sattributes processor in the presence of tail_sampling

### ISSUE THREAD

**Issue by gebn** [open-telemetry/opentelemetry-collector#8086]
Please see https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23870.

In short, `tail_sampling` must appear after `k8sattributes`, as the former obviates the context used by the latter for enrichment. This is a documentation fix, PR incoming.

**Comment by atoulme** [comment]
It looks like this has been addressed, ok to close?

**Comment by gebn** [comment]
I don't think it has been, https://github.com/open-telemetry/opentelemetry-collector/pull/8085 was the PR. I can reopen?
